File: Extensions/Stateflows.Extensions.PlantUml/Events/PlantUmlInfoRequest.cs
using Stateflows.Common;

namespace Stateflows.Extensions.PlantUml.Events
{
    [NoTracing]
    [NoImplicitInitialization]
    [Event(nameof(PlantUmlInfoRequest))]
    public sealed class PlantUmlInfoRequest : IRequest<PlantUmlInfo>
    { }
}

File: Transport/@stateflows/common/src/events/plant-uml-info.request.ts
import { StateflowsEvent } from "../decorators/stateflows-event";
import { PlantUmlInfo } from "./plant-uml-info";
import { Request } from "./request";

@StateflowsEvent("Stateflows.Extensions.PlantUml.Events.PlantUmlInfoRequest, Stateflows.Extensions.PlantUml")
export class PlantUmlInfoRequest extends Request<PlantUmlInfo> { }